style.css

```diff
@@ -28,15 +23,621 @@ ul {
 
 a {
   text-decoration: none;
-  color: white;
 }
 
-a:hover {
-  color: #333;
+p {
+  margin-bottom: 1rem;
+  font-size: 1.4rem;
+  line-height: 1.8rem;
+}
+
+hr {
+  color: lightgray;
+}
+
+img {
+  width: 100%;
 }
 
 /*
     ==============================================
                 ****** GLOBAL ******
     ==============================================
 */
+
+/* Navbar section */
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  padding: 5px;
+}
+
+.btn-nav {
+  padding: 10px;
+  margin: 0 5px;
+  background-color: white;
+  border: none;
+  font-size: 1rem;
+}
+
+.btn-nav:hover {
+  background-color: #f7f7f7;
+}
+
+.nav-link {
+  color: black;
+  font-weight: bold;
+}
+
+.nav-link:hover {
+  text-decoration: underline;
+}
+
+.img-nav {
+  height: 2.2rem;
+}
+
+.btn {
+  background-color: #567b95;
+  padding: 10px;
+  border-radius: 5px;
+  color: white;
+}
+
+.btn:hover {
+  background-color: #326891;
+}
+
+/* Main section */
+
+.container {
+  margin: 0 auto;
+  width: 1200px;
+  text-align: center;
+}
+
+.flex-container {
+  display: flex;
+  width: 100%;
+}
+
+.header {
+  width: 1200px;
+  margin: 0 auto;
+}
+
+.heading {
+  width: 600px;
+  margin: 0 auto;
+  margin-top: 40px;
+}
+
+h1 {
+  margin: 20px auto;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-weight: bold;
+  font-style: italic;
+}
+
+.social-networks {
+  margin-top: 20px;
+}
+
+.social-net {
+  display: inline-block;
+  width: 30px;
+  height: 30px;
+  color: black;
+  background-color: lightgray;
+  border-radius: 50%;
+  padding: 3px;
+  text-align: center;
+  padding-top: 5px;
+  margin: 5px;
+  border: 1px solid lightgray;
+}
+
+.social-net:hover {
+  background-color: white;
+  border: 1px solid lightgray;
+}
+
+.icon {
+  color: black;
+  font-size: 1.5rem;
+  margin: 5px;
+}
+
+.heading-img {
+  width: 100%;
+  height: auto;
+}
+
+figure {
```

Comment:
### Aim to remain DRY
- [ ] This rule seems to be repeating more than once and makes you appear as if you are repeating yourself. Do you mind merging these ones so as to make the codebase look neat?

**The repeating rules are on lines `#122` & `#156`**